### PR TITLE
Wilsond descriptions

### PR DESCRIPTION
**Pull request by donwilson-rezare**
I have applied descriptions to the fields in the Geospatial repository that don't have a description.

Also added an example to landUseActivityType and to spatialResourceType.  I'm not sure if I am doing these correctly and so pushing this now before I do more.

### CODE REVIEW

**Review by cookeac — CHANGES_REQUESTED** [review]
@donwilson-rezare all the description updates look fine.
There is an issue with the "examples".

Unfortunately, "examples" is the JSON Schema keyword for an array of examples.
In the `\url-schemes` folder are two json OpenAPI schema files. These are currently OpenAPI version 3.0.0 (actually 3.0.3, but who's counting). Unfortunately, OpenAPI 3.0 _does not support_ the "examples" keyword.

A solution would be to update these two OpenAPI specification files to version 3.1.0.
However, this version does not support the `"nullable": true` construct that appears 35 times in 19 files in this repository. Instead you have to specify the "type" as an array e.g. `"type": ["string", "null"]`. This would seem to be feasible, but for types specified via `$ref` one must go back to the referenced file and make the change there.

In short, my recommendation is to remove the "examples" changes from this PR, and then do a separate PR which replaces nullable with a null type option, updates the url-schemes examples to 3.1.0, adds the `"$schema": "https://json-schema.org/draft/2020-12/schema"` dialect tag to each resource file, and adds the examples.

See also: https://www.openapis.org/blog/2021/02/16/migrating-from-openapi-3-0-to-3-1-0

I have added issue #68 for this.
